src/apps/startups/models.py

```diff
@@ -4,14 +4,36 @@
 from django.db import models
 from django.utils import timezone
 
-from apps.users.models import Founder
+from apps.users.models import Founder, User
+
+
+class Project(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    project_name = models.CharField("Project Name", max_length=64)
+    PROJECT_TYPE = (("P", "Project"), ("I", "Idea"))
+
+    type = models.CharField(
+        max_length=1,
+        choices=PROJECT_TYPE,
+        blank=True,
+        default="S",
+        help_text="Project type",
+    )
+    created_at = models.DateTimeField()
+    description = models.TextField("Bio", max_length=425, blank=True, null=True)
```

Comment:
The verbose name "Bio" might be confusing since the field is named ```description```. Ensure the verbose name aligns with the field's purpose. If the field is meant to hold a project description, consider using a more appropriate verbose name like "Description"

Reply:
Fixed in 300042ae1be1b767a40a218997f6a254c08f2771